server/app/views/applicant/ApplicantUpsellFragment.html

```diff
@@ -0,0 +1,26 @@
+<html th:lang="${#locale.language}">
```

Comment:
this should be <html th:attr="lang=${preferredLanguage.code()}">

The PR that adds this on other pages is still under review, sorry I didn't have a model PR for it yet!

Reply:
oops sorry it should be th:attr="lang=${preferredLanguage.code()}">

see https://github.com/civiform/civiform/pull/7492

Reply:
Did you mean to provide a code snippet?

Reply:
yep. I think I successfully added it in a follow up comment